kk/views/hearing.py

```diff
@@ -80,6 +80,17 @@ def follow(self, request, pk=None):
         # return success
         return response.Response({'status': 'You follow a hearing now'}, status=status.HTTP_201_CREATED)
 
+    @detail_route(methods=['post'])
+    def unfollow(self, request, pk=None):
+        hearing = self.get_object()
```

Comment:
Since we have `permission_classes = (permissions.IsAuthenticatedOrReadOnly,)` there is already a check for authentication of post request.